libr/core/cmd.c

```diff
@@ -455,7 +455,10 @@ R_API int r_core_run_script (RCore *core, const char *file) {
 			free (out);
 		}
 	} else if (r_parse_is_c_file (file)) {
-		char *out = r_parse_c_file (file);
+		const char *aa = r_config_get (core->config, "asm.arch");
```

Comment:
too much spaguetti to get arch/os/bits. this info is already inside core->anal. just take it from there